src/commonMain/kotlin/com.adamratzman.spotify/SpotifyLogger.kt

```diff
@@ -0,0 +1,10 @@
+/* Spotify Web API - Kotlin Wrapper; MIT License, 2019; Original author: Adam Ratzman */
+package com.adamratzman.spotify
+
+expect class SpotifyLogger(enabled: Boolean) {
+    var enabled: Boolean
+
+    fun logInfo(message: String)
+    fun logWarning(message: String)
+    fun logError(fatal: Boolean, message: String?, throwable: Throwable?)
+}
```

Comment:
Why is it a platform specific implementation? There isn't any need for that.

Reply:
Because output is platform specific

Reply:
Aren't we using `println` for that? It's available in common.

Reply:
there’s no guarantee colors will work though 😉

Slightly related, the implementation of Throwable differs on each platform - Throwable/JVM allows printing the stack trace

Reply:
Yeah just found that issue with the `Throwable`.
So in my opinion, the logic should be in common and there should be a `expect fun Throwable.printStackTrace()` instead of the whole Logger.

And by the way, there isn't even a guarantee that the colors work on the JVM side, but I'm fine with that.